assets/css/modules-admin.css

```diff
@@ -50,3 +50,20 @@
 .chosen-container-multi .chosen-choices li.search-field input[type="text"] {
 	min-height: 25px !important;
 }
+
+/* Fixes action menu display in WP 6+ */
+.student-action-menu .components-popover {
+	opacity: 1;
+}
+
+.student-action-menu .components-popover__content {
+	left: -185px;
```

Comment:
Looks good to me now 👍 I was thinking, instead of using a fixed pixel value, should we use something like this `transform:translateX(-90%);`? In case the width varies or it's not clear where 185 came from.

Reply:
Good idea, thanks @Imran92 !  Done ✅